Classes/Controller/Ajax/AbstractMessengerAjaxController.php

```diff
@@ -10,6 +11,57 @@
 
 abstract class AbstractMessengerAjaxController
 {
+    public function getDemand(array $uids, ?string $moduleSignature, string $searchTerm): array
+    {
+        $demand = [
+            'likes' => [],
+            'uids' => [],
+        ];
+
+        // only if we have a list of uids
+        if (!empty($uids)) {
+            $demand['uids'] = $uids;
+        }
+        // only if we have a search term
+        if (strlen($searchTerm) > 0 && $moduleSignature != '') {
+            $demandedFields = [];
+            switch ($moduleSignature) {
```

Comment:
@compaoreh338 Créer une méthode $this->getDemandedFields et l'utiliser ici. Récupérer $moduleSignature dans la méthode getDemandedFields depuis les paramètres GET